psql_authnz.conf

```diff
@@ -8,5 +8,5 @@ respawn
 
 script
   . /opt/psql-authnz.env/bin/activate
-  exec su -s /bin/sh -c 'exec "$0" "$@"' postgres -- python /opt/psql-authnz/psql_authnz.py > /var/log/psql_authnz.log
+  exec su -s /bin/sh -c 'exec "$0" "$@"' postgres -- python /opt/psql-authnz/psql_authnz.py
```

Comment:
Quick question. When using the "-s" option, I thought it opened a whole new shell. Are Environment Variables declared in bash carried over in sh? I imagine they must be, otherwise this script wouldn't have any env vars to work with.
http://man7.org/linux/man-pages/man1/su.1.html